workshop/content/30-python/70-advanced-topics/200-pipelines/4000-build-stage.md

```diff
@@ -73,6 +73,7 @@ Then we add that stage to our pipeline (`pipeline.add_stage(deploy);`). A `Stage
 Now that we have added the code to deploy our application, all that's left is to commit and push those changes to the repo.
 
 ```
+git add .
 git commit -am "Add deploy stage to pipeline" && git push
```

Comment:
`git commit -a` already does git add . Why is the extra add required?

Reply:
The ``` -am``` flag not working. ``` -a -m``` worked for me.